### PR TITLE
Cleaned up docs of last references to .options

### PR DESCRIPTION
**Pull request by philippjfr**
Replaces any last usages of ``.options`` in the user guides and the apps.

### CODE REVIEW

File: examples/user_guide/14-Large_Data.ipynb [hunk 1]
```diff
@@ -544,11 +532,24 @@
...
    "name": "python",
-   "pygments_lexer": "ipython3"
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.6"
   }
  },
  "nbformat": 4,
- "nbformat_minor": 1
+ "nbformat_minor": 2
```
Comment (jlstevens) [inline examples/user_guide/14-Large_Data.ipynb]: Just need to clear out this metadata.

Comment (philippjfr) [reply examples/user_guide/14-Large_Data.ipynb]: Why?

Comment (philippjfr) [reply examples/user_guide/14-Large_Data.ipynb]: This metadata is always there, your script doesn't strip this.

Comment (philippjfr) [reply examples/user_guide/14-Large_Data.ipynb]: Ah sorry, missed the bit above.

Comment (philippjfr) [reply examples/user_guide/14-Large_Data.ipynb]: Fixed.

**Comment by jlstevens** [comment]
Thanks for this! The last commit is just stripping out notebook metadata so I'll just merge.

**Pull request merged**